float64 start_map_x
float64 start_map_y
float64 goal_map_x
float64 goal_map_y
---
bool isPlanLegal
